### PR TITLE
Add ifNot_ to macro

### PR DESCRIPTION
**Pull request by libraryaddict**

### CODE REVIEW

File: src/combat.ts [hunk 1]
```diff
@@ -356,6 +398,20 @@ export class Macro {
...
 
+  /**
+   * Create a new macro with an "if !" statement.
+   * @param condition The BALLS condition for the if statement.
+   * @param ifFalse Continuation if the condition is false.
+   * @returns {Macro} This object itself.
+   */
+  static ifNot_<T extends Macro>(
+    this: Constructor<T>,
+    condition: Parameters<T["ifNot_"]>[0],
```
Comment (horrible-little-slime) [inline src/combat.ts:409]: Rather than using Parameters, can we name the type that we use in if and ifn't somewhere?

Comment (horrible-little-slime) [reply src/combat.ts:409]: Other than that, I'm into it.

Comment (libraryaddict) [reply src/combat.ts:409]: I'm not really that experienced in typescript to do that

**Comment by horrible-little-slime** [comment]
Sniped you, I'm afraid

**Pull request closed without merging**